### PR TITLE
Implemented The frontend for superadmin

### PR DESCRIPTION
**Pull request by Vayras**
frontend for superadmin 
runs on  route /superadmin 
sphinx superadmin

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Here's the [figma file](https://miro.com/app/board/uXjVNQOK7Zc=/?moveToWidget=3458764570814683543&cot=14).

File: frontend/app/src/config/ModeDispatcher.tsx
Comment (kevkevinpal) [inline frontend/app/src/config/ModeDispatcher.tsx:1]: please revert this change

File: frontend/app/src/config/host.ts
Comment (kevkevinpal) [inline frontend/app/src/config/host.ts:1]: please revert this change

**Comment by Vayras** [comment]
Sure will fix and implement these changes

**Comment by Vayras** [comment]
I have made some changes also the table pagination works fine with mock bounties data
a
let me know what you think

**Comment by Vayras** [comment]
Can you maybe list some changes that you see that you think really needs to be fixed? That would be a great help thanks.

**Comment by kevkevinpal** [comment]
I'm seeing the eslint check and prettier check failing

**Comment by ecurrencyhodler** [comment]
@kevkevinpal let's hold off on merging this until we get the design right as well.

File: frontend/app/src/pages/index.tsx [hunk 1]
```diff
@@ -15,12 +15,17 @@ import { Modals } from './Modals';
...
+
 
 const modeDispatchPages: Record<AppMode, () => React.ReactElement> = {
   community: () => (
     <MainLayout header={<PeopleHeader />}>
       <TokenRefresh />
       <Switch>
+        <Route path="/superadmin">
+            <SuperAdmin />
+        </Route>
```
Comment (kevkevinpal) [inline frontend/app/src/pages/index.tsx:28]: there is already a route below named `/admin` lets use that one

File: frontend/app/src/pages/MainLayout.tsx [hunk 2]
```diff
@@ -1,22 +1,33 @@
...
   const c = colors['light'];
+  const location = useLocation(); 
+
   return (
-    <div className="app" style={{ background: c.background }}>
-      {header}
-      {children}
-    </div>
+    <>
+      {location.pathname === '/superadmin' ? (
```
Comment (kevkevinpal) [inline frontend/app/src/pages/MainLayout.tsx:21]: lets change this route to just `/admin`

File: frontend/app/src/pages/superadmin/Header/index.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          </NavWrapper>
+          <AlternateWrapper>
+            <Flex>
+              <ButtonWrapper>
+                <ArrowButton>
+                  <img src={arrowback} alt=""  />
+                </ArrowButton>
+                <ArrowButton><img src={arrowforward} alt=""  /></ArrowButton>
+              </ButtonWrapper>
+              <Month>01 Oct-31 Dec 2023</Month>
```
Comment (kevkevinpal) [inline frontend/app/src/pages/superadmin/Header/index.tsx:44]: lets not hardcode this month, lets use moment to figure out the current date and then set these dynamically

File: frontend/app/src/pages/superadmin/index.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+const Container = styled.body`
+  height: 100vh; /* Set a fixed height for the container */
+  overflow-y: auto; /* Enable vertical scrolling */
+`;
+
+
+export const SuperAdmin = () => {
+
+  console.log("super admin",bounties);
```
Comment (kevkevinpal) [inline frontend/app/src/pages/superadmin/index.tsx:16]: we can remove this log

File: package.json
Comment (kevkevinpal) [inline package.json:1]: we dont need this file we already have a package.json in `/frontend/src`

File: yarn.lock
Comment (kevkevinpal) [inline yarn.lock:1]: we dont need this `yarn.lock`

**Comment by Vayras** [comment]
@ecurrencyhodler how About this? let me know what you think so I can raise the PR
WorkDone

**Comment by ecurrencyhodler** [comment]
Amazing work vayras! Thank you so much for your attention to detail. I have a few items that I noticed which should be quick fixes.

Can we change:

- Satoshies -> Satoshis
- Capitalize: Bounties assigned -> Bounties Assigned
- Capitalize: OPEN, PAID, ASSIGNED
- Add "..." to the names in the bounty section if it exceed a certain length.
- "Bounties Paid" (next to 100%) -> Completed

**Comment by ecurrencyhodler** [comment]
Also can you address the eslint and prettier issues?

**Comment by Vayras** [comment]
I have added the commit , I might have tagged the wrong issue number , apologies for that

**Comment by ecurrencyhodler** [comment]
@Vayras can you do one more screenshot of your updates? Then we can do one more code review.

**Comment by Vayras** [comment]
> @Vayras can you do one more screenshot of your updates? Then we can do one more code review.

Yeah sure

**Comment by Vayras** [comment]
Screenshot 2023-12-11 232940

**Comment by Vayras** [comment]
@ecurrencyhodler here's the screenshot , for the issue , for now I have centered the whole thing for a better user experience on ultrawide monitors , let me know your perspective.

**Comment by ecurrencyhodler** [comment]
@Vayras what text font are you using? 

And are the status font bold or not bold?

**Comment by ecurrencyhodler** [comment]
Thanks for addressing my other comments!

**Comment by Vayras** [comment]
> @Vayras what text font are you using? 
> 
> And are the status font bold or not bold?

Barlow , same as the figma file , status font is set to bold

**Comment by Vayras** [comment]
> Thanks for addressing my other comments!

You're welcome, let me know if there's an update for me , or if you want any changes

**Comment by ecurrencyhodler** [comment]
Okay for some reason it doesn't look the same as the figma. 

But let's go ahead with a code review @kevkevinpal or @elraphty

**Comment by kevkevinpal** [comment]
had issues updating accidentally closed so its reopened here sorry for any inconvenience 

https://github.com/stakwork/sphinx-tribes/pull/1080
